aiohttp/client_reqrep.py

```diff
@@ -168,7 +168,8 @@ def __init__(self, method, url, *,
                  proxy=None, proxy_auth=None,
                  timer=None, session=None, auto_decompress=True,
                  ssl=None,
-                 proxy_headers=None):
+                 proxy_headers=None,
+                 traces=[]):
```

Comment:
Let's not use mutables as defaults. These may cause some nasty bugs we could easily avoid.

Reply:
true, changed to `None`